plugins/jobs/src/js/data/JobModel.ts

```diff
@@ -80,74 +69,8 @@ function isJobQueryArg(arg: any): arg is JobQueryArgs {
   return arg.id !== undefined;
 }
 
-// Sort and filtering
-function sortJobs(
-  jobs: Job[],
-  sortBy: SortOption = "ID",
-  sortDirection: SortDirection = "ASC"
-): Job[] {
-  jobs.sort((a, b) => {
-    let result;
-
-    switch (sortBy) {
-      case "ID":
-        result = compareJobById(a, b);
-        break;
-      case "STATUS":
-        result = compareJobByStatus(a, b);
-        break;
-      case "LAST_RUN":
-        result = compareJobByLastRun(a, b);
-        break;
-      default:
-        result = 0;
-    }
-
-    const direction = sortDirection === "ASC" ? 1 : -1;
-    return result * direction;
-  });
-
-  return jobs;
-}
-
-function compareJobById(a: Job, b: Job): number {
-  return a.id.localeCompare(b.id);
-}
-
-function compareJobByStatus(a: Job, b: Job): number {
-  return (
-    JobStates[a.scheduleStatus].sortOrder -
-    JobStates[b.scheduleStatus].sortOrder
-  );
-}
-
-function compareJobByLastRun(a: Job, b: Job): number {
-  const statusA = a.lastRunStatus ? a.lastRunStatus.status : "N/A";
-  const statusB = b.lastRunStatus ? b.lastRunStatus.status : "N/A";
-
-  return JobStatus[statusA].sortOrder - JobStatus[statusB].sortOrder;
-}
-
-function filterJobsById(
-  jobs: MetronomeJobResponse[],
-  filter: string | null
-): MetronomeJobResponse[] {
-  if (filter === null) {
-    return jobs;
-  }
-
-  return jobs.filter(({ id }) => id.indexOf(filter) !== -1);
-}
-
-function filterJobsByNamespace(
-  jobs: MetronomeJobResponse[],
-  namespace: string | null
-): MetronomeJobResponse[] {
-  if (namespace === null) {
-    return jobs;
-  }
-
-  return jobs.filter(({ id }) => id.startsWith(namespace));
+function isJobsQueryArg(arg: any): arg is JobsQueryArgs {
```

Comment:
I'd say `arg` isn't `any`, shall we use `object` instead? Or do we really expect getting `any` here?

Reply:
For these user-defined type guards you need to use any AFAIK

Reply:
From the documentation 🤷‍♂️ 
```js
function isFish(pet: Fish | Bird): pet is Fish {
    return (<Fish>pet).swim !== undefined;
}
```

Reply:
ah okay, good to know 👍